lib/net/ber/ber_parser.rb

```diff
@@ -10,7 +10,8 @@ module Net::BER::BERParser
     5 => :null,
     6 => :oid,
     10 => :integer,
-    13 => :string # (relative OID)
+    13 => :string, # (relative OID)
+    139 => :array
```

Comment:
This is most confusing part for me, I was hoping you could shed some light on it. This is the only way I could get it too work, I will have another look at it.